src/wasm/wasm-validator.cpp

```diff
@@ -577,6 +582,7 @@ void FunctionValidator::visitAtomicWait(AtomicWait* curr) {
 }
 
 void FunctionValidator::visitAtomicNotify(AtomicNotify* curr) {
+  shouldBeTrue(getModule()->memory.exists, curr, "Memory operations require a memory");
```

Comment:
The bulk memory operations should also have this check.

Reply:
Adding, thanks.